intro-HPC/ch_gpu_gent.tex

```diff
@@ -0,0 +1,260 @@
+
+\begin{itemize}
+    \item brief motivation on why you want to join the pilot;
+    \item list of software applications (that support GPUs) you would like to use
+    \item example job that can be used as a test case (scripts, input data + short description)
+\end{itemize}
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+\section{Submitting jobs}
+\label{sec:gpu_ugent_jobs}
+
+To submit jobs to the HPC-UGent GPU cluster nicknamed \lstinline|joltik| during the pilot phase, first use:
+
+\begin{prompt}
+%\shellcmd{module swap cluster/.joltik}
+\end{prompt}
+
+Please note the dot (\lstinline|.|) in the module name (which makes this \lstinline|cluster| module a hidden module).
+
+Then use the familiar \lstinline|qsub|, \lstinline|qstat|, etc.\ commands, taking into account the guidelines outlined
+in section~\ref{sec:gpu_ugent_resources}.
+
+\strong{Note that you will only be able to submit jobs to the \lstinline|joltik| cluster if you are a member of the
+pilot group (see \ref{sec:gpu_ugent_pilot}).} If not, you will see the following error message:
+
+
+\begin{prompt}
+salloc: error:
+Job submit/allocate failed: User's group not permitted to use this partition
+\end{prompt}
+
+\subsection{Interactive jobs}
+\label{sec:gpu_ugent_interactive_jobs}
+
+To interactively experiment with GPUs, you can submit an interactive job using \lstinline|qsub -I| (and request
+one or more GPUs, see section~\ref{sec:gpu_ugent_resources}).
+
+Note that due to a bug in Slurm you will currently not be able to be able to interactively use
+MPI software that requires access to the GPUs. If you need this, please contact use via \hpcinfo.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+\section{Hardware}
+
+See \url{https://www.ugent.be/hpc/en/infrastructure/overzicht.htm}.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Requesting (GPU) resources %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+\section{Requesting (GPU) resources}
+\label{sec:gpu_ugent_resources}
+
+There are 2 main ways to ask for GPUs as part of a job:
+
+\begin{itemize}
+  \item Either as a node property (similar to the number of cores
+  per node specified via \lstinline|ppn|) using \lstinline|-l nodes=X:ppn=Y:gpus=Z| (where the \lstinline|ppn=Y| is optional),
+  or as a separate resource request (similar to the amount of memory) via \lstinline|-l gpus=Z|.
+  Both notations give exactly the same result.
+  The \lstinline|-l gpus=Z| is convenient is you only need one
+  node and you are fine with the default number of cores per GPU.
+  The \lstinline|-l nodes=...:gpus=Z| notation is required if you want to run with full control or in multinode cases like MPI jobs.
+  In you do not specify the number of GPUs, you get by default 1 GPU.
```

Comment:
euhm, users still have to ask the `-l gpus` or `:gpus` part. it's not that you always get a gpu by default (or am i reading too much into this?)

Reply:
You wrote this, not me. ;)

I'll reword it.